buildkite/bazel-central-registry/bcr_presubmit.py

```diff
@@ -482,12 +482,19 @@ def main(argv=None):
             bazelci.eprint("No target module versions detected in this branch!")
         pipeline_steps = []
         for module_name, module_version in modules:
+            previous_size = len(pipeline_steps)
+
             configs = get_task_config(module_name, module_version)
             add_presubmit_jobs(module_name, module_version, configs.get("tasks", {}), pipeline_steps)
             configs = get_test_module_task_config(module_name, module_version)
             add_presubmit_jobs(module_name, module_version, configs.get("tasks", {}), pipeline_steps, is_test_module=True)
+
+            if len(pipeline_steps) == previous_size:
+                error("No pipeline steps generated for %s@%s. Please check the configuration." % (module_name, module_version))
```

Comment:
is it now an error to not have any tasks in presubmit.yml? was it an error before?

Reply:
Yes, previously you'll get a green build, which isn't what's expected, see https://github.com/bazelbuild/bazel-central-registry/pull/1656#discussion_r1533558361

Reply:
I understand that this catches e.g. misspelled `tasks`, but what about when someone actually doesn't want any tasks? I've gotten similar questions before. Is our answer just "no, you must always have at least one task"?

Reply:
> Is our answer just "no, you must always have at least one task"?

I think so? There must be some targets that are buildable from the module? Otherwise, it's useless.

Reply:
Ping